modules/ethereum/src/lib.rs

```diff
@@ -326,6 +328,25 @@ pub trait PruningStrategy: Default {
 	fn pruning_upper_bound(&mut self, best_number: u64, best_finalized_number: u64) -> u64;
 }
 
+/// ChainTime represents the runtime on-chain time
+pub trait ChainTime: Default {
```

Comment:
Why do we need the `Default` bound?

Reply:
An instance of `ChainTime` is needed for the Dispatchables. Reasoning is that the `Default` bound would save everyone from having to implement `new()`. Or am I missing something here?